README.md

```diff
@@ -11,4 +11,8 @@ To start the services, run `docker-compose up -d`.
 TODO 
 
 Steps to generate a topology from the archetypes
-Launch a topology + run it on the cluster
+Launch a topology ( (1) via Flux (2) via Java) + run it on the cluster
+  - (a) Inject seeds
+  - (b) Run crawl
+
+If elasticsearch / kibana: Explain how to create index + mappings (or link to documentation)
```

Comment:
The focus will be on using URLFrontier and not ES. The injection is done separately on the command line